resources/policies/runtimeintegrity/ttps.yaml

```diff
@@ -837,23 +837,24 @@
   prefilter: [PE]
 
 - list: fs_discovery_cmds
-  items: [mount, df]
+  items: [mount, df, tree, find]
 
 - macro: home_dir_arg
   condition: sf.proc.args endswith '/home' or sf.proc.args endswith '/home/'
 
+# partially from https://github.com/SigmaHQ/sigma/blob/master/rules/linux/lnx_file_and_directory_discovery.yml
 - rule: File and Directory Discovery
   desc: enumerate files, directories and volume information
   condition: sf.opflags = EXEC and
-             ((sf.proc.name = ls and home_dir_arg) or
-              sf.proc.name in (fs_discovery_cmds))
+             ((sf.proc.name = ls and (home_dir_arg or sf.proc.args contains '-R')) or
+               sf.proc.name in (fs_discovery_cmds))
   action: [tag]
   priority: high
   tags: [mitre:T1083]
   prefilter: [PE]
 
 - list: net_discovery_cmds
-  items: [netstat, ss]
+  items: [netstat, ss, lsof, last, w, who]
```

Comment:
1. I moved the lists and macros as requested. 
2. Re. w and who commands: we had an overlap between `user_discovery_cmds` and `net_discovery_cmds` which both contained `w` and `who` commands. The result was that these commands would have caused two alerts tagged with T1033 and T1049. This would be inline with MITRE since the correct association depends on the context (previous activities). Since we don't have that context in a SysFlow policy I decided to separate  `user_discovery_cmds` and `net_discovery_cmds` to avoid double-tagging:

```
- list: user_discovery_cmds
  items: [w, who, whoami, id, last]
...
- list: net_discovery_cmds
  items: [netstat, ss, lsof]
```
Let me know what you think.